src/main/java/org/openhab/binding/zwave/internal/protocol/ZWaveController.java

```diff
@@ -107,7 +107,7 @@ public class ZWaveController {
 
     private ZWaveInclusionController inclusionController = null;
     private int defaultWakeupPeriod = 0;
-    private int maxbatterywakeduration = 5;
+    private int maxAwakePeriod = 5;
```

Comment:
To avoid confusion, I'd suggest to remove the `= 5` from the end of this - it doesn't do anything.

Reply:
Done

Again just patterning after the defaultWakeupPeriod

`private int maxAwakePeriod;`